form.html

```diff
@@ -0,0 +1,141 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+      <header class="main-header">
+      <a href="#" class="main-header__logo">
+        <img class="main-header__logo-img" src="" alt="Логотип Кэт энерджи." width: 33.32px; height: 37.61px; >
+        <img src="" alt="Логотип Кэт энерджи." class="main-header__logo-img" >
+      </a>
+      <nav class="main-nav">
+        <button class="main-nav__toggle" type="button"><span class="visually-hidden">Открыть меню.</span>
+        </button>
+      </nav>
+      <div class="main-nav__wrapper">
+        <ul class="site-list">
+<li class="site-list__item">
+  <a href="#" class="site-list__item-link">
+    Главная
+  </a>
+</li>
+<li class="site-list__item">
+  <a href="#" class="site-list__item-link">
+Каталог продукции
+  </a>
+</li>
+<li class="site-list__item">
+  <a href="#" class="site-list__item-link">
+  Подбор программы
+  </a>
+</li>
+  </ul>
+      </div>
+    </header>
+
+<main class="main-form">
+   <section class="main-form-container">
+    <h1 class="main-form__title">
+        Подбор программы
+    </h1>
+    <p class="main-form_subtitle">
+        Заполните анкету, и мы подберем
+программу питания для вашего кота
+     </p>
+    <form action="index.html"  method="post" class="main-form__form-ance form">
```

Comment:
```suggestion
    <form action="index.html"  method="post" class="main-form__form-ance form">
```
поправь action
форма одна
внутри могут быть fieldset'ы